Java/core/src/main/java/com/amazon/randomcutforest/tree/PointerTree.java

```diff
@@ -58,36 +58,27 @@ protected String toString(double[] doubles) {
         return Arrays.toString(doubles);
     }
 
-    void updateBox(Node nodeReference) {
-        checkState(enableCache, " incorrect invocation");
-        BoundingBox rightBox = nodeReference.getRightChild().getBoundingBox();
-        BoundingBox leftBox = nodeReference.getLeftChild().getBoundingBox();
-        nodeReference.setBoundingBox(leftBox.getMergedBox(rightBox));
+    @Override
+    void setCachedBoxes(Node node, AbstractBoundingBox<double[]> savedBox) {
```

Comment:
Shouldn't this be `setCachedBox` (singular)?

Reply:
Done.